fastlane_core/spec/env_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe FastlaneCore do
+  describe FastlaneCore::Env do
```

Comment:
A test for nil would be great too, since an unset variable is different from a nil variable in ruby